source/agora/consensus/EnrollmentManager.d

```diff
@@ -451,6 +483,23 @@ public class EnrollmentManager
         catch (Exception ex)
         {}
     }
+
+    /***************************************************************************
+
+        Function that reverse Hash by converting it into Byte array
+
+        Params:
+            data = hash of target to reverse
+
+        Return:
+            Reverse bytes and return to the new ubyte[]
+
+    ***************************************************************************/
+
+    private static ubyte[] reverseUbytes (ref const Hash data) @trusted nothrow
+    {
+        return data[].dup.reverse;
```

Comment:
What is this function for? The commit message doesn't explain it. Why is the byte array reversed?

Also, this should use a buffer instead. Like a `static ubyte[]`, set its length to `data[]` and then copy the data over with `buffer[] = data[]`.

Reply:
> What is this function for? The commit message doesn't explain it. Why is the byte array reversed?

Added commit message description.

Also, this should use a buffer instead. Like a static ubyte[], set its length to data[] and then copy the data over with buffer[] = data[].

I added a buffer.